### PR TITLE
Coldblooded Updates

### PR DESCRIPTION
**Pull request by Geevies**
* Made Unathi properly coldblooded. They now don't generate their own bodyheat. In exchange, they use less nutrition and thirst to continue functioning.
* Removed Unathi always spawning with sandals. Pick sandals in the loadout menu, they were made free. Otherwise, pick other footwear, they will now be equipped automatically.
* Coldblooded human mobs now regenerate stamina slower when cold, even losing stamina when very cold. And regenerate stamina faster when warm.
* Added heatpacks and coldpacks, they increase or decrease the temperature you receive from an environment by five Kelvin. Does not work in space.
* Added padded winter coats that spawn with heatpacks and coldpacks in them.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/modules/clothing/suits/hoodies.dm [hunk 1]
```diff
@@ -245,3 +245,22 @@
...
+
+/obj/item/clothing/suit/storage/hooded/wintercoat/heated/Initialize()
+	..()
+	var/obj/item/clothing/accessory/heatpack/pack = new(src)
+	attach_accessory(null, pack)
+
+/obj/item/clothing/suit/storage/hooded/wintercoat/cooled
+	name = "padded coat"
+	desc = "A heavy jacket made from animal furs. This one appears to have special pockets for a temperature pack."
+
```
Comment (Nero-07) [inline code/modules/clothing/suits/hoodies.dm]: Should be "a special pocket for a temperature pack" or " special pockets for temperature packs", no? Same above.

File: code/modules/clothing/under/accessories/accessory.dm [hunk 2]
```diff
@@ -757,4 +758,27 @@
...
+	icon_state = "heat_pack"
+	icon_override = 'icons/mob/ties.dmi'
+	slot_flags = SLOT_TIE
+	w_class = ITEMSIZE_SMALL
+	slot = "over"
+	temperature = 5
+
+/obj/item/clothing/accessory/coldpack
+	name = "cold pack"
+	desc = "This pack contains chemicals that, when activated, provide a substantial but safe amount of cooling."
```
Comment (Nero-07) [inline code/modules/clothing/under/accessories/accessory.dm]: "... substanstial, but safe, amount ..." I think.

File: html/changelogs/geeves - coldbloods.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+author: Geeves
+
```
Comment (Nero-07) [inline html/changelogs/geeves - coldbloods.yml]: Not your name :p

**Pull request closed without merging**